app/views/user/show.erb

```diff
@@ -1,6 +1,7 @@
 <div class="container">
-  <section class="page-header">
-  <h1><%= profile.username %></h1>
+  <section class="row page-header">
+    <div class="col-md-1 main-profile-img"><%= gravatar_tag profile.current_user.avatar_url, size: 90 %></div>
```

Comment:
When you visit someone else's profile , you will see your profile image instead of their profile image